sdk/management/README.md

```diff
@@ -79,14 +79,16 @@ With above configuration, `azure` client can be authenticated by following code:
 
 ```java
 AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
-TokenCredential credential = new DefaultAzureCredentialBuilder()
+TokenCredential credential = new EnvironmentCredentialBuilder()
     .authorityHost(profile.environment().getActiveDirectoryEndpoint())
     .build();
 Azure azure = Azure
     .authenticate(credential, profile)
     .withDefaultSubscription();
 ```
 
+The sample code assumes global Azure. Please change `AzureEnvironment.AZURE` variable if otherwise.
```

Comment:
do we need to update the migration guide and auth.md for this?

Reply:
I've add a note and sample in AUTH.md as well.

Reply:
thanks Weidong